### PR TITLE
Added revised dataset files to dataset folder 

### PR DESCRIPTION
**Pull request by rishicjain**
This is a upgraded version of the PR :- https://github.com/microsoftgraph/dataconnect-solutions/pull/264

### CODE REVIEW

File: Datasets/data-connect-dataset-contact.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+| Profession |	string |	The contact's profession. |	No |	None |
+| SpouseName |	string |	The name of the contact's spouse. |	No |	None |
+| Surname |	string	| The contact's surname. |	No |	None |
+| Title |	string |	The contact's title. |	No |	None |
+| YomiCompanyName |	string |	The phonetic Japanese company name of the contact. |	No	| None |
+| YomiGivenName |	string |	The phonetic Japanese given name (first name) of the contact. |	No |	None |
+| YomiSurname |	string |	The phonetic Japanese surname (last name) of the contact. |	No |	None |
+| ODataType |	string |	Data type of the current folder. |	No |	None |
+| puser	 | string |	User id. | 	No |  	None |
+| ptenant |	strinf  | 	Tenant id. |	No |  	None |
```
Comment (pushkarprotik) [inline Datasets/data-connect-dataset-contact.md:80]: typo in "string" line L80

File: Datasets/data-connect-dataset-groupdetails.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+4df826af4e22","displayName":"Groupa20c15f8","expirationDateTime":null,"groupTypes":["Unified"],"isAssignableToRole":null,"mail":"groupa20c15f8@contosotest4.onmicrosoft.com","mailEnabled":true,"mailNickname":"groupa20c15f8","membershipRule":null,"membershipRuleProcessingState":null,"onPremisesDomainName":null,"onPremisesLastSyncDateTime":null,"onPremisesSyncEnabled":null,"preferredDataLocation":null,"preferredLanguage":null,"proxyAddresses":["SPO:SPO_70e575f2-820c-4da1-832b-2118c56a4be8@SPO_537d6c63-efb6-4922-8643-17921eb1b0dd","SMTP:groupa20c15f8@contosotest4.onmicrosoft.com"],"renewedDateTime":"2018-07-16T12:18:55Z","resourceProvisioningOptions":[],"securityEnabled":false,"securityIdentifier":"S-1-12-1-4029065122-1292943853-1474286271-4107509874","theme":null,"visibility":"Public","ODataType":"#microsoft.graph.group","pObjectId":"f026a7a2-c1ed-4d10-bfd2-df5772a0d3f4","ptenant":"537d6c63-efb6-4922-8643-17921eb1b0dd"}
```
Comment (pushkarprotik) [inline Datasets/data-connect-dataset-groupdetails.md:129]: I am seeing some of the sample jsons like this one having red marked lines, these could be improper jsons formed due to maybe a newline.
Please validate all sample data where-ever these red lined jsons are present as they may be malformed jsons

File: Datasets/data-connect-dataset-groupdetails.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+{"id":"9be41eaa-3486-44ae-b8ab-534034ea8452","deletedDateTime":null,"classification":null,"createdDateTime":"2021-04-30T21:46:26Z","description":null,"displayName":"Test-For-AdminPortal","expirationDateTime":null,"groupTypes":[],"isAssignableToRole":null,"mail":"Test-For-AdminPortal@contosotest4.onmicrosoft.com","mailEnabled":true,"mailNickname":"Test-For-AdminPortal","membershipRule":null,"membershipRuleProcessingState":null,"onPremisesDomainName":null,"onPremisesLastSyncDateTime":null,"onPremisesSyncEnabled":null,"preferredDataLocation":null,"preferredLanguage":null,"proxyAddresses":"SMTP:Test-For-AdminPortal@contosotest4.onmicrosoft.com"],"renewedDateTime":"2021-04-30T21:46:26Z","resourceProvisioningOptions":[],"securityEnabled":true,"securityIdentifier":"S-1-12-1-2615418538-1152267398-1079225272-1384442420","theme":null,"visibility":null,"ODataType":"#microsoft.graph.group","pObjectId":"9be41eaa-3486-44ae-b8ab-534034ea8452","ptenant":"537d6c63-efb6-4922-8643-17921eb1b0dd"}
```
Comment (pushkarprotik) [inline Datasets/data-connect-dataset-groupdetails.md:141]: please check the json formatting here

File: Datasets/data-connect-dataset-message1.md
Comment (pushkarprotik) [inline Datasets/data-connect-dataset-message1.md:1]: Not able to render this file, will check in local

File: Datasets/data-connect-dataset-outlookgroupconversations.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+| ReplyTo |	string |	The email addresses to use when replying. |	No |	None |
+| Body |	string	| The UniqueBody in raw format. |	No |	None |
+| ReceivedDateTime | 	datetime |	The date and time the message was received. |	Yes |	Date |
+| Sender |	string |	The account that is actually used to generate the message. |	No |	None |
+| SentDateTime |	datetime |	The date and time the message was sent. |	Yes |	Date |
+| Subject |	string |	The subject of the message. |	No |	None |
+| ToRecipients |	string |	The To recipients for the message. |	No |	None |
+| UniqueBody |	string |	The part of the body of the message that is unique to the current message. It is in text format. |	No |	None |
+| WebLink |	string	| The URL to open the message in Microsoft Outlook web app. |	No |	None |
+| oDataType |	string |	Data type of the current folder. |	No	| None |
```
Comment (pushkarprotik) [inline Datasets/data-connect-dataset-outlookgroupconversations.md:67]: O should be capital: "ODataType"

Comment (rishicjain) [reply Datasets/data-connect-dataset-outlookgroupconversations.md]: Adressed in the latest commit

File: Datasets/data-connect-dataset-groupdetails.md (hunk 3 above)
Comment (rishicjain) [reply Datasets/data-connect-dataset-groupdetails.md]: Adrressed in the latest commit

File: Datasets/data-connect-dataset-groupdetails.md (hunk 2 above)
Comment (rishicjain) [reply Datasets/data-connect-dataset-groupdetails.md]: Adressed all the relevant files in the latest commit

File: Datasets/data-connect-dataset-contact.md (hunk 1 above)
Comment (rishicjain) [reply Datasets/data-connect-dataset-contact.md]: Fixed in latest commit

**Pull request merged**